specs/pvs/corla.pvs

```diff
@@ -863,6 +934,8 @@ BEGIN
 % @description The audited contest determines the number of ballots
 % that must be examined and verified during the RLA.
   number_of_ballots_to_audit: [audited_contest -> nat]
+
+% @todo kiniry Still need to model core RLA algorithm(s).
 END rlas
 
 %+END audits
```

Comment:
line 878/971 ff: 
"They frequently, but not always, contain an interpretation of all contests on a ballot."

better might be: "an interpretation of voter choices in all contests on a ballot"